system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_endpoint_test.clj

```diff
@@ -14,162 +17,245 @@
-           :update-type "ADD_TO_EXISTING"}
-          400
-          ["/concept-ids array is too short: must have at least 1 elements but instance has 0 elements"]
-
-          "0 collection ids"
-          {:collection-ids []
-           :update-field "SCIENCE_KEYWORDS"
-           :update-type "ADD_TO_EXISTING"}
-          400
-          ["/collection-ids array is too short: must have at least 1 elements but instance has 0 elements"]
-
-          "Missing short-name"
-          {:collection-ids [{:version "V1"}]
-           :update-field "SCIENCE_KEYWORDS"
-           :update-type "ADD_TO_EXISTING"}
-          400
-          ["/collection-ids/0 object has missing required properties ([\"short-name\"])"]
-
-          "Missing version"
-          {:collection-ids [{:short-name "ABC"}]
-           :update-field "SCIENCE_KEYWORDS"
-           :update-type "ADD_TO_EXISTING"}
-          400
-          ["/collection-ids/0 object has missing required properties ([\"version\"])"]
-
-          "Missing update field"
-          {:concept-ids ["C1", "C2", "C3"]
-           :update-type "ADD_TO_EXISTING"}
-          400
-          ["object has missing required properties ([\"update-field\"])"]
-
-          "Missing update type"
-          {:concept-ids ["C1", "C2", "C3"]
-           :update-field "SCIENCE_KEYWORDS"}
-          400
-          ["object has missing required properties ([\"update-type\"])"]
-
-          "Invalid update type"
-          {:concept-ids ["C1", "C2", "C3"]
-           :update-field "SCIENCE_KEYWORDS"
-           :update-type "REPLACE"}
-          400
-          ["/update-type instance value (\"REPLACE\") not found in enum (possible values: [\"ADD_TO_EXISTING\",\"CLEAR_FIELD\",\"CLEAR_ALL_AND_REPLACE\",\"FIND_AND_REMOVE\",\"FIND_AND_REPLACE\"])"]
-
-          "Missing update value"
-          {:concept-ids ["C1", "C2", "C3"]
-           :update-field "SCIENCE_KEYWORDS"
-           :update-type "FIND_AND_REPLACE"}
-          400
-          ["An update value must be supplied when the update is of type FIND_AND_REPLACE"]
-
-          "Missing find value"
-          {:concept-ids ["C1", "C2", "C3"]
-           :update-field "SCIENCE_KEYWORDS"
-           :update-type "FIND_AND_REPLACE"
-           :update-value "X"}
-          400
-          ["A find value must be supplied when the update is of type FIND_AND_REPLACE"])))
+  (let [token (grant-permissions-create-token)]
+    (testing "Request body validation"
+      (are3 [body status-code error-messages]
+            (let [response (ingest/bulk-update-collections "PROV1" body {:token token})
+                  {:keys [status errors]} response]
+              (is (= status-code status))
+              (is (= error-messages errors)))
+
+            "Missing collection identifiers"
+            {:update-field "SCIENCE_KEYWORDS"
+             :update-type "ADD_TO_EXISTING"}
+            400
+            ["instance failed to match exactly one schema (matched 0 out of 2)"
```

Comment:
This is from JSON schema validation. The 2 lines below come back and explain the problem. This is also from another PR.